apps/e2e/src/indexer-client/rewardsQueriesTests.ts

```diff
@@ -0,0 +1,28 @@
+import { IndexerClient } from '@vertex-protocol/indexer-client';
+import { runWithContext } from '../utils/runWithContext';
+import { RunContext } from '../utils/types';
+import { TimeInSeconds } from '@vertex-protocol/utils';
+import { prettyPrint } from '../utils/prettyPrint';
+
+async function rewardsQueriesTests(context: RunContext) {
```

Comment:
What about `getRewards`, `getTakerRewards`, `getClaimVrtxMerkleProofs`, `getClaimFoundationRewardsMerkleProofs`, `getSonicPoints`, and `getSonicPointsLeaderboard`?